SU2_CFD/include/solvers/CTurbSolver.hpp

```diff
@@ -249,6 +251,24 @@ class CTurbSolver : public CSolver {
                           CNumerics *visc_numerics,
                           CConfig *config) final;
 
+  /*!
+   * \brief Set the solution using the Freestream values.
+   * \param[in] config - Definition of the particular problem.
+   */
+  inline void SetFreeStream_Solution(const CConfig *config) final {
+    SU2_OMP_FOR_STAT(omp_chunk_size)
+    for (unsigned long iPoint = 0; iPoint < nPoint; iPoint++){
+      nodes->SetSolution(iPoint, Solution_Inf);
+    }
+  }
```

Comment:
Awesome, more generic.
But pleeeeeeaaaaseeeee do not allocate memory with new/delete (see my love letter to containers) use a std::array of MAXNVAR or even a static array.

Reply:
Ok it's `su2double Solution_Inf[MAXNVAR]` now